.travis.yml

```diff
@@ -1,11 +1,31 @@
 language: objective-c
-osx_image: xcode8.3
+osx_image: xcode9.2
```

Comment:
This is defined in the matrix below.  Do we need to duplicate here?

Reply:
Not sure! Removed and we'll see...